github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/context/publishers/GitHubCommentPublisherDslContext.java

```diff
@@ -0,0 +1,49 @@
+package org.jenkinsci.plugins.github.pullrequest.dsl.context.publishers;
+
+import hudson.model.Result;
+
+import org.jenkinsci.plugins.github.pullrequest.GitHubPRMessage;
+import org.jenkinsci.plugins.github.pullrequest.publishers.impl.GitHubPRCommentPublisher;
+import org.jenkinsci.plugins.github.pullrequest.utils.PublisherErrorHandler;
+import org.jenkinsci.plugins.github.pullrequest.utils.StatusVerifier;
+
+import javaposse.jobdsl.dsl.Context;
+
+public class GitHubCommentPublisherDslContext implements Context {
+
+    public static final GitHubPRCommentPublisher DEFAULT_PUBLISHER = new GitHubPRCommentPublisher();
+
+    private PublisherErrorHandler errorHandler;
+
+    private GitHubPRMessage message = GitHubPRCommentPublisher.DEFAULT_COMMENT;
+
+    private StatusVerifier verifier;
+
+    public void comment(String comment) {
+        message = new GitHubPRMessage(comment);
+    }
+
+    public GitHubPRCommentPublisher getPublisher() {
+        return new GitHubPRCommentPublisher(message, verifier, errorHandler);
+    }
+
+    public void markPublishErrorsAsFailure() {
```

Comment:
these are all the new dsl methods, can you just comment on the ones whose name needs to be changed  here since they aren't all covered in the test dsl script.